tests/integration/helpers/cluster.py

```diff
@@ -2997,6 +3000,7 @@ def logging_pulling_images(**kwargs):
                 os.mkdir(self.minio_dir)
                 if self.minio_certs_dir is None:
                     os.mkdir(os.path.join(self.minio_dir, "certs"))
+                    os.mkdir(os.path.join(self.minio_dir, "certs", "CAs"))
```

Comment:
This change isn't related to the added logging - it's just to ensure that the `CAs` folder is created with the privileges which allow removing that folder on a repeated test run.